tests/System.Memory.Polyfill.Tests/System.Memory.Polyfill.Tests.csproj

```diff
@@ -1,13 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\..\tools\common.props" />
   <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
Is listing netcoreapp2.1 TFM here redundant or required (for Windows only)?

Reply:
I changed it in the new PR. This actually did not work in the CI. I changed it to use netcoreapp2.0 to force the fallback path.